internal/stack/compose.go

```diff
@@ -188,3 +184,56 @@ func withIsReadyServices(services []string) []string {
 	}
 	return allServices
 }
+
+func dockerComposeStatus() ([]ServiceStatus, error) {
+	var services []ServiceStatus
+	// query directly to docker to avoid load environment variables (e.g. STACK_VERSION_VARIANT) and profiles
+	containerIDs, err := docker.ContainerIDsWithLabel(fmt.Sprintf("%s=%s", projectLabelDockerCompose, DockerComposeProjectName))
```

Comment:
Nit. Pass the key and the value to `ContainerIDsWithLabel` so the caller doesn't need to be aware of the format of the docker flag?
```suggestion
	containerIDs, err := docker.ContainerIDsWithLabel(projectLabelDockerCompose, DockerComposeProjectName)
```